PowerPointLabs/PowerPointLabs/ThisAddIn.cs

```diff
@@ -733,7 +733,10 @@ private void UpdateELearningPane(int selectedSlidesCount)
             if (selectedSlidesCount > 0)
             {
                 taskpane.eLearningLabMainPanel1.HandleSlideChangedEvent();
-                taskpane.eLearningLabMainPanel1.HandleELearningPaneSlideSelectionChanged();
+                if (elearningLabPane.Visible == true)

```

Comment:
Should this check also include line 735? Why is line 735  outside of this check?

Reply:
This check should not include line 735. This is because when slide selection is changed, two things will happen:
1. Check if previous slide contains any unsynced items. If so, we will prompt user to save changes.
2. Load e-learning lab with new items from current slide animations pane.

The first event should happen regardless of whether e-learning lab is visible. For example, if user opens e-learning lab and performs some changes, then closes it. A message should still prompt to user to sync.